astropy/cosmology/tests/test_core.py

```diff
@@ -30,6 +31,7 @@
 ##############################################################################
 # SETUP / TEARDOWN
 
+PY_LT_311 = sys.version_info < (3, 11)
```

Comment:
I feel like this should be defined in `astropy.utils.compat` somewhere if you need this check in `time` too. And spelling out `PYTHON_LT_3_11` would make it easier to grep when we bump minversion and stuff. Thanks!

Reply:
✅